src/app/components/registration-form/registration-form.component.scss

```diff
@@ -3,4 +3,7 @@
 } 
 .form-error{
     border: 1px solid red;
+}
```

Comment:
If you are using the same class multiple times, it might be good to try and put this higher in the hierarchy. It's probably going to be in more places.